app/content/carto-user-manual/maps/map-settings.md

```diff
@@ -195,3 +199,21 @@ Select the Google Maps Basemap menu to see different basemap options. They inclu
 - **Positron**, **Voyager** and **Dark Matter**: new versions that has been developed for Google Maps.
 
 ![Map basemap google](/img/cloud-native-workspace/maps/map_basemap_google_maps.png)
+
+
+#### Amazon Location
+
+Select _Amazon Location_ in the menu to see the options to use a base map from the [Amazon Location Service](https://docs.aws.amazon.com/location/index.html).
+
+To use an Amazon Location maps, you will need to introduce some credentials from your Amazon Location Service account. Lear more about it in [this guide]((https://docs.aws.amazon.com/location/latest/developerguide/using-maps.html)): 

```

Comment:
This is the only link that I cannot open. Maybe it is because of the double parenthesis ¿?